traffic_stats/influxdb_tools/sync/sync_test.go

```diff
@@ -0,0 +1,230 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/influxdata/influxdb/models"
+	. "github.com/smartystreets/goconvey/convey"
```

Comment:
I generally agree with @PSUdaemon and @dewrich . Looking at goconvey, it seems to violate Separation of Concerns, blending infrastructure and build tools with testing code. Seems like those ought to be separate as possible; it should be possible to replace infrastructure tools or testing libraries independently. 

Don't get me wrong, the goconvey webpage stuff looks really useful, I'd certainly support a similar infrastructure tool which was independent of the test framework/library (ideally something integrating seamlessly with the native `go test`).

Also, in general, Go has built-in testing, is what goconvey adds really worth the cost of an external dependency?

All that said, if everyone else feels it will help testing _that_ much, I'm not extremely opposed. I vote "-0".